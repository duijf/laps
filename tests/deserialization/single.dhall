let Laps = ../../package.dhall

in  [ { name = "foo"
      , shortDesc = "Run the foo"
      , startOrder =
          Laps.single
            { alias = "foo"
            , executable =
                Laps.program
                  { program = "foo", arguments = [ "--bar", "--baz" ] }
            , nixEnv = None Laps.NixEnv
            , watchExtensions = [] : List Text
            }
      }
    ]
