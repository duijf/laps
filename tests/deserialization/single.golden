[ Command
    { name = "foo"
    , shortDesc = "Run the foo"
    , startOrder = Single
        ( Unit
            { executable = P
                ( Program
                    { program = "foo"
                    , arguments =
                        [ "--bar"
                        , "--baz"
                        ]
                    }
                )
            , alias = "foo"
            , nixEnv = Nothing
            , watchExtensions = []
            }
        )
    }
]